docs/usage/source/index.md

```diff
@@ -0,0 +1,36 @@
+---
+id: source
+title: "Source"
+sidebar_position: 1
+---
+
+While deploying Botkube, you can specify which [executors](../../configuration/executor) you want to enable.
```

Comment:
"which sources" 😄

Reply:
I think you should ignore index.md for now :) I was waiting for kubernetes extraction, but now I will update it without waiting kubernetes.